dss/operations/secrets.py

```diff
@@ -36,7 +39,7 @@ def get_secretsmanager_prefix(args):
     "list",
     arguments={
         "--stage": dict(
-            required=False, help="the stage for which secrets should be listed"
+            required=True, help="the stage for which secrets should be listed"
```

Comment:
The code in `dss/*` and `scripts/*` uses the stage defined in the env var `DSS_DEPLOYMENT_STAGE`. Passing in a different stage as an argument to dss-ops scripting may result it conflicts or other undefined behavior.

This may not be the case for secrets handling, but even so it would be preferable to maintain consistency with the rest of the dss operational scripting.

Reply:
Good catch - will make this a non-required flag. The cod already looks for the stage name in the `DSS_DEPLOYMENT_STAGE` env var if the flag is undefined.